client-ui/src/app/components/lineage/lineage-graph/lineage-graph.component.ts

```diff
@@ -126,13 +126,11 @@ export class LineageGraphComponent implements OnChanges, AfterViewInit {
     })
   }
 
-  private highlightAttrLinAndImp(attrLinGraph: AttributeGraph, attrImpGraph: AttributeGraph) {
+  private highlightAttrLinAndImp(attrLinGraph: AttributeGraph | undefined, attrImpGraph: AttributeGraph) {
```

Comment:
if the second argument cannot be undefined, then the second argument should be first:

`
highlightAttrLinAndImp(attrImpGraph: AttributeGraph, attrLinGraph?: AttributeGraph)
`